### DISCUSSION TITLE
Timezone used for vacation start/stop times

### DISCUSSION THREAD

**Discussion by billpitz** [postfixadmin/postfixadmin#792] [General]
I have users across many timezones. My server is set to UTC time. What appears to happen is that when a user sets their vacation message start/stop, it is done using UTC (this is fine). The value they enter is what is saved in the database, and that seems to properly determine when the vacation message starts/stops based on the server time. However, when they go back to edit the vacation message, the time that is displayed in the browser is adjusted to their local timezone. If the make a change to the time and save again, the value that is saved to the database (and interpreted as UTC) is the value that was in their own timezone.  If they come back to edit a third time, the time is now adjusted by the offset from UTC an additional time.

How can I set the vacation page to use either UTC or the user's browser timezone across the board? I don't care which one it is, it just needs to be consistent all the time.

**Comment by DavidGoodwin** [comment]
I don't see any nice way of getting a date/time from a user which includes a timezone - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local

So perhaps we'd need a drop down to add a timezone selector?

🤷🏾‍♂️
